src/test/java/HelloWorldTest.kt

```diff
@@ -0,0 +1,11 @@
+import org.junit.Test
+
+class HelloWorldTest {
```

Comment:
Diese Datei gehört nicht in `src/test/java` sondern in `src/test/kotlin` - damit das funktioniert, müsst ihr die POM noch ein bisschen erweitern (siehe Vorlesung).